server/database/models/rating.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
+module.exports = (sequelize, DataTypes) => {
+  const Rating = sequelize.define('Rating', {
+    rating: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    articleId: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      reference: {
+        model: 'Article',
+        key: 'id',
+      },
+    },
+  }, {});
```

Comment:
I think you should index the `articleId` and `userId` to be `unique` to ensure that the combination of userid and articleid are unique and cannot be duplicated

For instance, 
```
 {
    indexes: [
      {
        unique: true,
        fields: ['articleId', 'userId'],
      },
    ],
  }
```

Reply:
The above code should be in the second argument of the model declaration